cmd/istioctl/testdata/four-route-rules.yaml

```diff
@@ -0,0 +1,39 @@
+type: route-rule
+name: test-v1
+spec:
+  destination: productpage.default.svc.cluster.local
+  precedence: 1
+  route:
+  - tags:
+      version: v1
+    weight: 25
+---
+type: route-rule
+name: test-v2
+spec:
+  destination: reviews.default.svc.cluster.local
+  precedence: 1
+  route:
+  - tags:
+      version: v2
+    weight: 25
+---
+type: route-rule
+name: test-v3
+spec:
+  destination: ratings.default.svc.cluster.local
+  precedence: 1
+  route:
+  - tags:
+      version: v3
+    weight: 25
+---
+type: route-rule
```

Comment:
add a destination policy here too? I want to make sure we handle a variety of kinds?